src/main/kotlin/com/vanniktech/maven/publish/BaseMavenPublishPlugin.kt

```diff
@@ -36,6 +38,17 @@ internal abstract class BaseMavenPublishPlugin : Plugin<Project> {
     }
   }
 
+  private fun checkGradleVersion(p: Project) {
```

Comment:
You can use `VersionNumber.parse` and then use `>=` to compare.

Reply:
Ah, thanks, I was wondering if such a thing existed!